COPY/etc/manageiq/postgresql.conf.d/01_miq_overrides.conf

```diff
@@ -51,6 +56,9 @@ hot_standby = on
 # QUERY TUNING
 #------------------------------------------------------------------------------
 
+random_page_cost = 4
+effective_cache_size = 6GB
```

Comment:
ManageIQ appliances only ship with 6GB of RAM, so this would be too high I think.

Also the default is 4GB so we're already over what your recommendation (1/2 of total memory) would be.

Reply:
Dropped to the recommended 4608MB.